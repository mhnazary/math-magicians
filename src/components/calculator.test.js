import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { prettyDOM } from '@testing-library/dom';

describe('Calculator', () => {
  test('calculator is rendering', () => {
    const calculator = render(<calculator />);
    expect(prettyDOM(calculator.container)).toMatchSnapshot();
  });
});
