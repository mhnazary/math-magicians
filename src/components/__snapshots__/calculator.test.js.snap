// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator calculator is rendering 1`] = `
"[36m<div>[39m
  [36m<calculator />[39m
[36m</div>[39m"
`;
